### PR TITLE
Don't show cached OAI DC if disabled, refs #12272

### PR DESCRIPTION
**Pull request by mcantelon**
Added check to see if XML caching is enabled before showing cached results.

### CODE REVIEW

File: plugins/arOaiPlugin/lib/arOaiPluginComponent.class.php [hunk 1]
```diff
@@ -148,6 +148,12 @@ public static function cachedMetadataExists($resource, $metadataPrefix)
     return file_exists(QubitInformationObjectXmlCache::resourceExportFilePath($resource, $format, true));
   }
 
+  public static function shouldDisplayCachedMetadata($resource, $metadataPrefix)
```
Comment (sbreker) [inline plugins/arOaiPlugin/lib/arOaiPluginComponent.class.php]: Wondering if this should be renamed to make it more clear what it does?  Would just  'displayCachedMetadataIfExists()' work?  Or checkDisplayCachedMetadata()?

I am finding 'shouldDisplayCachedMetadata()' to be unclear.

**Review by sbreker — CHANGES_REQUESTED** [review]
I've added one comment.  Let me know what your thoughts are on that.

File: plugins/arOaiPlugin/lib/arOaiPluginComponent.class.php (hunk 1 above)
Comment (mcantelon) [reply plugins/arOaiPlugin/lib/arOaiPluginComponent.class.php]: Yeah, I like the latter... more clear.

**Review by sbreker — APPROVED** [review]
Looks great!  Thanks @mcantelon !

**Pull request merged**